tests/rptest/tests/transactions_test.py

```diff
@@ -604,6 +605,107 @@ def consume_records(consumer, count):
 
         consume_records(consumer, 200)
 
+    @cluster(num_nodes=3)
+    def check_sequence_table_cleaning_after_eviction_test(self):
+        segment_size = 1024 * 1024
+        topic_spec = TopicSpec(partition_count=1, segment_bytes=segment_size)
+        topic = topic_spec.name
+
+        # make segments small
+        self.client().create_topic(topic_spec)
+
+        producers_cnt = 20
```

Comment:
`producers_cnt` / `message_count` let's stick to something one (I prefer `_count`)